.circleci/config.yml

```diff
@@ -154,8 +154,11 @@ jobs:
           name: Authenticate with npm
           command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/atlantis/.npmrc
       - run:
-          name: Publishing
-          command: npm run ci:release
+          name: Release Packages
+          command: |
+            if git log --max-count=1 --pretty=%s | grep -q "chore(release): Publish"; then
+              echo npm run ci:release
```

Comment:
As discussed, you may want a negaton on the condition, and don't forget to remove the `echo` from the command itself.